pwsh-extension-dev.code-workspace

```diff
@@ -0,0 +1,493 @@
+{
+  "folders": [
+    {
+      "name": "Client",
+      "path": "."
+    },
+    {
+      "name": "Server",
+      "path": "../PowerShellEditorServices"
+    }
+  ],
+
+  "extensions": {
+    "recommendations": [
+      "davidanson.vscode-markdownlint",
+      "dbaeumer.vscode-eslint",
+      "editorconfig.editorconfig",
+      "ms-azure-devops.azure-pipelines",
```

Comment:
```suggestion
```

Since the ADO pipelines are now internal only we don't need to recommend this extension.

Reply:
👍